index.html

```diff
@@ -82,63 +117,95 @@ <h2>-</h2>
             provide.
           </p>
         </div>
-        <img src="./img/mobile/4.webp" alt="" /><img
-          src="./img/mobile/5.webp"
-          alt=""
-        /><img src="./img/mobile/6.webp" alt="" />
-        <h1>BUY ONLINE NOW & GET 10% OFF !</h1>
-        <button>Shop Now</button>
+        <div class="mobile-main-pics">
+          <img src="./img/mobile/4.webp" alt="4" /><img
+            src="./img/mobile/5.webp"
+            alt="5"
+          /><img src="./img/mobile/6.webp" alt="6" />
```

Comment:
You should specify explicit width and height on  <img> to avoid cumulative content shift. Reference: https://www.aleksandrhovhannisyan.com/blog/setting-width-and-height-on-images/